site/scripts/generate-blocks-data.ts

```diff
@@ -0,0 +1,12 @@
+import { writeFileSync } from "fs";
+import path from "path";
+import { readBlocksFromDisk } from "../src/lib/blocks";
+
```

Comment:
Had to go with the approach of generating a json file that contains blocks data (similar to what we do for sitemap). This is because of the difficulty I faced in making the `api` access the `hub/` folder at runtime. I feel we can work with this for the main time since it's a temporary solution

Reply:
Yes, sounds good.

I'm assuming the hub folder is not accessible as it's outside of the `site` folder, where this application is built from.

Reply:
Yes. That's correct